qutebrowser/browser/webkit/webpage.py

```diff
@@ -283,6 +289,30 @@ def on_load_started(self):
         else:
             self.error_occurred = False
 
+    @pyqtSlot()
+    def inject_userjs(self, load):
+        """Inject user javascripts into the page.
+
+        param: The page load stage to inject the corresponding scripts
+               for. Support values are "start", "end" and "idle",
+               corresponding to the allowed values of the `@run-at`
+               directive in the greasemonkey metadata spec.
+        """
+        greasemonkey = objreg.get('greasemonkey')
+        url = self.currentFrame().url()
+        scripts = greasemonkey.scripts_for(url)
+
+        if load == "start":
+            toload = scripts.start
+        elif load == "end":
+            toload = scripts.end
+        elif load == "idle":
+            toload = scripts.idle
```

Comment:
Probably a good idea to throw an exception or log message in an else block here (to avoid 'toload not defined' errors below)

Reply:
This should never get called with anything else, and I think the exception would be clear enough FWIW.